### PR TITLE
Research completed for: shop/supermarket|Tesco Metro

### LINKED ISSUE #1751: Research needed for: shop/supermarket|Tesco Metro
#### :thinking: We need help deciding what to do with "shop/supermarket|Tesco Metro"

:point_right: Read the [CONTRIBUTING.md](https://github.com/osmlab/name-suggestion-index/blob/master/CONTRIBUTING.md) guide for more info, and feel free to ask questions on this issue!

### What's needed

First, do some research on `shop/supermarket|Tesco Metro` by searching Google and Wikipedia.

#### If it is an actual brand:

* Edit `config/canonical.json`:
  * Find the entry for `shop/supermarket|Tesco Metro`
  * Add tags for `brand:wikidata` and `brand:wikipedia`
  * Optionally add `"countryCodes"` or other tags
* `npm run build`
* Commit your changes and submit a pull request

#### If it is not an actual brand:

* Edit `config/filters.json` to add an expression that discards it
* Edit `config/canonical.json` to remove the entry for `shop/supermarket|Tesco Metro`
* `npm run build`
* Commit your changes and submit a pull request

#### If the situation is complicated:

* Leave a comment on this issue with what you find

:sparkles: Thanks!

_This is an autogenerated ticket_

### PR DESCRIPTION
**Pull request by AlbyIanna**
closes #1751

### CODE REVIEW

**Comment by ENT8R** [comment]
According to https://overpass-turbo.eu/s/CWR this seems to be restricted to Great Britain. Is that true?

**Comment by AlbyIanna** [comment]
it seems so, but I couldn't find anything that confirmed the hypothesis. If you want me to add it i'll do it though

File: config/canonical.json [hunk 1]
```diff
@@ -31382,6 +31382,8 @@
     "count": 152,
     "tags": {
       "brand": "Tesco Metro",
+      "brand:wikidata": "Q487494",
```
Comment (ENT8R) [inline config/canonical.json]: I also think that we should use Q57551648 instead of the generic Tesco entry...

File: config/canonical.json [hunk 2]
```diff
@@ -31382,6 +31382,8 @@
...
+      "brand:wikipedia": "en:Tesco",
```
Comment (ENT8R) [inline config/canonical.json]: ...and maybe it is better to remove the `brand:wikipedia` tag because the actual brand is Tesco Metro and not Tesco...

**Comment by ENT8R** [comment]
> If you want me to add it i'll do it though

Yes, please. I read it up on wikipedia and it seems to be GB-only

File: config/canonical.json (hunk 1 above)
Comment (AlbyIanna) [reply config/canonical.json]: right

File: config/canonical.json (hunk 2 above)
Comment (AlbyIanna) [reply config/canonical.json]: okey dokey

File: config/canonical.json [hunk 3]
```diff
@@ -31382,8 +31382,8 @@
     "count": 152,
     "tags": {
       "brand": "Tesco Metro",
-      "brand:wikidata": "Q487494",
-      "brand:wikipedia": "en:Tesco",
+      "brand:wikidata": "Q57551648",
+      "countryCodes": ["gb"],
```
Comment (ENT8R) [inline config/canonical.json]: This value does not belong into the tags object. You have to add it below the value of count:*

**Comment by good-praxis** [comment]
Too bad `Q57551648` doesn't have a wikipedia article attached, but thanks for your contribution!

**Pull request merged**